### PR TITLE
Update run.py (now compatible with py>=3.0 & TF.20)

### PR DESCRIPTION
**Pull request by manncodes**
Changes made to random seed import change, and python 3 no longer supports indexing of values of dictionary.

### CODE REVIEW

File: run.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 from numpy.random import seed, shuffle
 
-from tensorflow import set_random_seed
```
Comment (manncodes) [inline run.py:15]: tensorflow 2.0 moved the set_random_seed.py to tensorflow.random and renamed it to just set_seed.

File: run.py [hunk 2]
```diff
@@ -133,8 +133,9 @@ def hyper_tune_network(dataset_name, epochs, batch_size, batch_epoch_ratio, figu
                         'cnn_train_type']  # 'cnn_arch','learning_rate','fix_len','use_aug','dropout', 'optimizer','optimizer',
     #
 
+    cnns_arch_values = list(cnns_arch.values())
     best_params_train = dict(optimizer=optimizers[0], learning_rate=learning_rates[0],
-                             cnn_train_type=cnn_train_types[0], cnn_arch=cnns_arch.values()[0],
+                             cnn_train_type=cnn_train_types[0], cnn_arch=cnns_arch_values[0],
```
Comment (manncodes) [inline run.py:138]: Dictionary indexing no longer supported.

**Review by liorsi — APPROVED** [review]
Thanks! looks great
